core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -510,9 +511,16 @@
    * LOAD_STATUS SUCCESS
    */
   public static final String STORE_LOADSTATUS_SUCCESS = "Success";
+
+  /**
+   * Default batch for data read in Columnar format
```

Comment:
Is it 4096 rows ? As per the below code , 4096 should be row number ? 
 
int resultSize = scannedResult.numberOfOutputRows();
    if (resultSize < batchSize) {
      batchSize = resultSize;
    }